src/JsonPath.Tests/ParsingTests.cs

```diff
@@ -139,4 +139,19 @@ public void ParseInOpWithoutOptions(string path)
 	{
 		Assert.Throws<PathParseException>(() => JsonPath.Parse(path));
 	}
+
+
+
+	[TestCaseSource(nameof(SuccessCases))]
+	public void ParseRelativeStarts(string path)
+	{
+		path = $"@{path[1..]}"; // Turn the absolute path into a relative path
+		Assert.DoesNotThrow(() => JsonPath.Parse(path, new PathParsingOptions(){AllowRelativePathStart = true}));
+	}
```

Comment:
Nit: there should be one line between adjacent tests/methods (here, you have none, and above, you have 3)

Reply:
Sorry, fixed!